### ISSUE TITLE
Please add C++/WinRT Examples

### ISSUE THREAD

**Issue by dr-eck** [MicrosoftDocs/winrt-api#1859]
This code worked at one point:

        TextBlock tb;
        tb.Text(L"Hello, Windows 10!");
        tb.FontFamily(FontFamily(L"Times New Roman"));
        tb.FontSize(216);
        tb.FontStyle(FontStyle::Italic);
        tb.Foreground(SolidColorBrush(Color::Red()));
        tb.HorizontalAlignment(HorizontalAlignment::Center);
        tb.VerticalAlignment(VerticalAlignment::Center);

It no longer works, at least in a WinUI 3 UWP project.  Please give an example of C++/WinRT code that does work.  Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 27ad25f2-d06e-135c-240c-99c8a0cfa728
* Version Independent ID: eccc6b3a-5607-158c-2ed6-949293923f2d
* Content: [TextBlock Class (Windows.UI.Xaml.Controls) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textblock?view=winrt-19041)
* Content Source: [winrt-api-build/xml/Windows.UI.Xaml.Controls/TextBlock.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.UI.Xaml.Controls/TextBlock.xml)
* Product: **uwp**
* Technology: **winui**
* GitHub Login: @jwmsft
* Microsoft Alias: **jimwalk**

**Comment by stevewhims** [comment]
Hi, @dr-eck,

Note that there *is* a Windows::UI::Colors class (Colors, plural), which has member functions such as Red(). But the Windows::UI::Color struct (Color, single) *doesn't* have member functions such as Red(). You can construct a red Windows::UI::Color from ARGB values, but you need the Colors class for "::Red()". So changing the use of Color in your example to Colors, gives the following.

Assuming this StackPanel in your markup: 
<StackPanel x:Name="myStackPanel"/>

And assuming you've included everything you need to (here are some examples that we use):
```cppwinrt
#include <winrt/Windows.UI.Text.h>
#include <winrt/Windows.UI.Xaml.Controls.h>
#include <winrt/Windows.UI.Xaml.Media.h>
```

Then this builds and runs (you could include it in a click handler, to test it):

```cppwinrt
Windows::UI::Xaml::Controls::TextBlock tb;
tb.Text(L"Hello, Windows 10!");
tb.FontFamily(Windows::UI::Xaml::Media::FontFamily(L"Times New Roman"));
tb.FontSize(36);
tb.FontStyle(Windows::UI::Text::FontStyle::Italic);
tb.Foreground(Windows::UI::Xaml::Media::SolidColorBrush(Windows::UI::Colors::Red()));
tb.HorizontalAlignment(HorizontalAlignment::Center);
tb.VerticalAlignment(VerticalAlignment::Center);
myStackPanel().Children().Append(tb);
```

Thanks!
-Steve

**Comment by dr-eck** [comment]
Hi Steve,

Thanks for responding.  I figured out that the Windows namespace has the required functions.  With some help from Stack Overflow, I figured out that what caused my code to fail was that I had switched from the Windows namespace to the Microsoft namespace, and the "FontStyle", "Colors" and "HorizontalAlignment" had not yet been implemented in that.  Switching back to the Windows namespace fixed all that.  

The only remaining issue was that I had started the project using WinUI 3 code, so the Grid defined in XAML was based on the Microsoft namespace, and when I tried to append a Windows::UI::Xaml::Controls::TextBlock to a Microsoft::UI::Xaml::Controls::Grid, the compiler found them to be incompatible.  Sigh.  I'm hoping the required pieces will be added to the Microsoft namespace in the November release.
